.hadolint.yml

```diff
@@ -3,6 +3,7 @@ ignored:
   - DL3008
   - DL3018
   - DL3059
+  - DL3026
```

Comment:
instead of adding this for all global, i would suggest adding the ignore just in that dockerfile instead